cdap-app-fabric/src/main/java/co/cask/cdap/security/TokenSecureStoreRenewer.java

```diff
@@ -70,16 +63,49 @@
   private Long updateInterval;
 
   @Inject
-  TokenSecureStoreUpdater(YarnConfiguration hConf, CConfiguration cConf,
+  TokenSecureStoreRenewer(YarnConfiguration hConf, CConfiguration cConf,
                           LocationFactory locationFactory,
                           co.cask.cdap.api.security.store.SecureStore secureStore) {
```

Comment:
Since we removed twill SecureStore, no need to have now this as fully qualified class name.